ERROR    asyncio:base_events.py:1738 Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7f4d2cef8b20>
ERROR    asyncio:base_events.py:1738 Unclosed connector
connections: ['[(<aiohttp.client_proto.ResponseHandler object at 0x7f4d2ce3c2e0>, 285.543311933)]']
connector: <aiohttp.connector.TCPConnector object at 0x7f4d2cef8bb0>